src/status_im/ui/screens/profile/styles.cljs

```diff
@@ -44,6 +44,13 @@
    :justify-content :center
    :align-items     :center})
 
+(def profile-badge-edit
+  {:flex-direction  :column
+   :justify-content :center})
+
+(def modal-menu-center
```

Comment:
Please rename to `modal-menu` - style names should only reflect what they refer to, not how they style it.